apps/nestjs-api/src/con-profiles/con-profiles.service.ts

```diff
@@ -39,13 +40,38 @@ export class ConProfilesService {
   async findAll(filter: any = {}) {
     const persistedRecords = await this.api.getAllConProfiles(filter)
 
-    const entities: ConProfileEntity[] = persistedRecords.map((source) =>
+    let entities: ConProfileEntity[] = persistedRecords.map((source) =>
       this.mapper.fromPersistence(source)
     )
 
+    entities = this.whenMultipleProfilesExcludeDeactivatedOnes(entities)
+
     return entities
   }
 
+  whenMultipleProfilesExcludeDeactivatedOnes(profiles: ConProfileEntity[]) {
```

Comment:
```suggestion
  excludeDeactivatedProfilesOfMultipleProfiles(profiles: ConProfileEntity[]) {
```

Reply:
Actually, how about `selectLatestActiveProfilePerUser`. I think it's also important to explain that the function also ensures we pick maximum one profile per contact.

Reply:
Sounds good!